This patch came from in-house.  A patch to make -I imply -S was submitted
upstream, but rejected, so this is the next simplest way to work around
issues with when the site module is imported.

--- Python-3.5.6/Lib/test/test_site.py	2018-08-02 02:19:12.000000000 +0000
+++ Python-3.5.6/Lib/test/test_site.py	2019-01-07 04:41:13.489672678 +0000
@@ -456,7 +456,7 @@ class StartupImportTests(unittest.TestCa
     def test_startup_imports(self):
         # This tests checks which modules are loaded by Python when it
         # initially starts upon startup.
-        popen = subprocess.Popen([sys.executable, '-I', '-v', '-c',
+        popen = subprocess.Popen([sys.executable, '-I', '-S', '-v', '-c',
                                   'import sys; print(set(sys.modules))'],
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE)
@@ -465,8 +465,6 @@ class StartupImportTests(unittest.TestCa
         stderr = stderr.decode('utf-8')
         modules = eval(stdout)
 
-        self.assertIn('site', modules)
-
         # http://bugs.python.org/issue19205
         re_mods = {'re', '_sre', 'sre_compile', 'sre_constants', 'sre_parse'}
         # _osx_support uses the re module in many placs
